pkg/apis/aiven.nais.io/v1/aiven_application_types.go

```diff
@@ -54,6 +54,8 @@ type AivenApplicationSpec struct {
 	SecretName string `json:"secretName"`
 	// A Protected secret will not be deleted by the janitor even when not in use
 	Protected bool `json:"protected,omitempty"`
+	// Timestamp for personal time limited secret. Configured with ISO timestamp.
+	ExpiresAt string `json:"expiresAt,omitempty"`
```

Comment:
Skulle vi prøvd å validere feltet?
Se på `kubebuilder:validation:Type` her https://book.kubebuilder.io/reference/markers/crd-validation.html
Må antageligvis testes litt for å se at det virker slik vi tror.

Reply:
Du tenker som doc. Sier sammen denne?
kubebuilder:validation:Format

Can teste lit lokalt

Reply:
ser ut til at de som blitt lagt inn nå validerer riktig, har testet med lite forskjellige input. Men `RFC3339` formatet validerer ok som input.